pkg/controller/certificates/BUILD

```diff
@@ -46,7 +47,7 @@ filegroup(
     ],
     tags = ["automanaged"],
     visibility = [
-        "//pkg/controller:__pkg__",
+        "//" + vendored_path_prefix + "pkg/controller:__pkg__",
```

Comment:
ah, I finally understand what you're doing here. this seems like the best possible solution, but I wonder if anyone else has encountered this (fixing references in BUILD files when native go vendoring a bazel-native project).

though doesn't this mean that all of the references in these BUILD files are wrong? or is this needed because the visibility won't get fixed by gazelle/kazel?

Reply:
Most BUILD files for vendored code get regenerated just fine by gazelle/kazel.  As you say, it's the things that aren't generated that need manual fixes.